pkg/kapp/config/default.go

```diff
@@ -515,6 +515,18 @@ changeGroupBindings:
   # delay other resources with load balancer provisioning
   # - apiVersionKindMatcher: {kind: Service, apiVersion: v1}
 
+- name: change-groups.kapp.k14s.io/serviceaccount
+  resourceMatchers: &serviceAccountMatchers
+  - apiVersionKindMatcher: {kind: ServiceAccount, apiVersion: v1}
+
+- name: change-groups.kapp.k14s.io/appcr
```

Comment:
cr should not be in the name (since everything else is also a resource).

change-groups.kapp.k14s.io/appcr => change-groups.kapp.k14s.io/kapp-controller-app